### ISSUE TITLE
Ability to show or hide Posts and Pages in the main admin menu

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#416]
```
What functionality do you want to see?

Ability to show or hide from the main admin menu:

* Posts
* Pages
* Links

Do you have examples of similar functionality on other sites or in other
plugins? (Please share URLs below)

http://managewp.com/wordpress-admin-sidebar-remove-unwanted-items


```

Original issue reported on code.google.com by `ever...@fireproofsocks.com` on 23 Sep 2012 at 11:04

**Comment by GoogleCodeExporter** [comment]
```
Add checkboxes to the Global Settings page.
```

Original comment by `ever...@fireproofsocks.com` on 1 Jan 2013 at 12:53
- Added labels: **Milestone-Release0.9.7**

**Comment by GoogleCodeExporter** [comment]
```
Committed revision 646866.

Posts, Pages, Links, and Comments.  Will be released in 0.9.7
```

Original comment by `ever...@fireproofsocks.com` on 2 Jan 2013 at 4:02
- Changed state: **Fixed**
